include/gmsh.h

```diff
@@ -1,18 +1,23 @@
 #ifndef MPM_POINT_GEN_GMSH_H_
 #define MPM_POINT_GEN_GMSH_H_
 
+#include <array>
+#include <algorithm>
 #include <fstream>
 #include <iostream>
 #include <limits>
 #include <map>
 #include <memory>
 #include <utility>
 #include <vector>
+#include <cmath>
```

Comment:
`cmath` should be at the top see: https://cpp.cb-geo.com/header/order_includes.html

Reply:
forgot to clang format...